generators/generator-bot-enterprise-calendar/README.md

```diff
@@ -1,47 +1,39 @@
 # @microsoft/generator-bot-enterprise-calendar [![NPM version][npm-image]][npm-url]
 
-This template creates a bot configured to manage Office 365 calendars using the Microsoft Graph API. **[Learn more](https://aka.ms/EnterpriseCalendarBot)**
+A bot with the ability to interact with Office 365 Calendar using Microsoft Graph. [Learn more](https://aka.ms/EnterpriseCalendarBot)
 
-## What this template is for
+### Recommended use
 
-Use this template if you want to...
+- Create a sophisticated bot that enables customers to interact with Office 365 calendar using Microsoft Graph
+- Customize and extend sophisticated example dialogs, bot logic, language understanding and bot responses
 
-- Support managing Office 365 calendars using Microsoft Graph
-- Start from an advanced template including dialogs, language understanding, and language generation 
+### Included capabilities
 
-## Packages
+- Manage Office 365 calendars using Microsoft Graph
 
-This bot uses the [Azure Bot Framework component model](https://aka.ms/ComponentTemplateDocumentation) to extend its base functionality. The following packages come pre-installed:
-- [Microsoft.Bot.Components.Graph](https://www.nuget.org/packages/Microsoft.Bot.Components.Graph/)
+### Included packages
 
-## Supported Languages
+- [Microsoft Graph](https://www.nuget.org/packages/Microsoft.Bot.Components.Graph/)
```

Comment:
Do we need to add an npm badge similar to below for the Graph package as well?

Reply:
In general I think our package links need work.  but I don't want to crack that for R13